### ISSUE TITLE
Python 3.13

### ISSUE THREAD

**Issue by thewchan** [samuelcolvin/rtoml#82]
The PyO3 used here seems to not support python 3.13:

```
 --- stderr
    error: the configured Python interpreter version (3.13) is newer than PyO3's maximum supported version (3.12)
    = help: please check if an updated version of PyO3 is available. Current version: 0.21.2
    = help: set PYO3_USE_ABI3_FORWARD_COMPATIBILITY=1 to suppress this check and build anyway using the stable ABI
```

**Comment by LukasVik** [comment]
PyO3 is in this project currently pegged at 0.21.2 which does not support Python 3.13: https://github.com/samuelcolvin/rtoml/blob/main/Cargo.toml#L10

Python 3.13 support was added to PyO3 in version 0.22, with improved support in 0.23: https://pyo3.rs/main/changelog

I tried to naively bump the version (#83) but there are some code updates needed in rtoml for compilation to pass. I don't know enough about Rust and the details of PyO3/rtoml to fix them myself. Once it compiles a new release needs to be made.

@samuelcolvin Could you help in bumping PyO3 and making a new release? Python 3.13 was released in October and is starting to gain usage. Would be great if rtoml could support it.

**Comment by samuelcolvin** [comment]
Will do soon, I'd also be happy to review a PR.

**Comment by LukasVik** [comment]
@samuelcolvin Thank you, I really appreciate your work.

**Comment by samuelcolvin** [comment]
fixed by #86
